resources/core/values.yaml

```diff
@@ -71,11 +71,11 @@ global:
     dir: develop/
     version: e11a469a
   ui_api_layer:
-    dir: develop/
-    version: ef6d4c13
+    dir: pr/
+    version: PR-2317
```

Comment:
I assume this is for testing purposes only? We need a separate PR with image bump.

Reply:
You are right. It will be changed before merging.